src/headerChainSync/HeaderChainChunk.js

```diff
@@ -1,61 +0,0 @@
-class HeaderChainChunk {
```

Comment:
Let's keep this header chain stuff for a while. I believe it will be useful pretty soon when we work on DAPI SPV